elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1435,12 +1495,22 @@ protected void delFromCollection(
             final Collection removedBidrectional = CollectionUtils
                     .disjunction(Collections.singleton(this.getObject()), originalBidirectional);
 
-            toDelete.checkFieldAwareDeferPermissions(
-                    UpdatePermission.class,
-                    inverseField,
-                    removedBidrectional,
-                    originalBidirectional
-            );
+            if (toDelete.isNewlyCreated()) {
```

Comment:
This should never happen afaik. @aklish while this sort of thing would be _technically_ possible, should we disallow it outright?